src/child/containers/App.js

```diff
@@ -17,9 +15,16 @@ require('script!../../../node_modules/moment/moment.js');
 require('script!../../../node_modules/BitFlux/node_modules/bootstrap/js/dropdown.js');
 require('script!../../../node_modules/d3fc/dist/d3fc.bundle.min.js');
 require('script!../../../node_modules/BitFlux/dist/bitflux.js');
+const DevTools = require('./devTools/DevTools.js');
+
+const contextMenu = (e) => {
+    if (process.env.NODE_ENV === 'production') {
+        e.preventDefault();
+    }
+};
 
 const App = ({ code, name }) => (
-    <div className="main">
+    <div className="main" onContextMenu={contextMenu}>
```

Comment:
Can the OpenFin config (`contextMenu: false`) be used for this?

https://openfin.co/application-config/